log/example/log_example.go

```diff
@@ -29,9 +29,9 @@ func main() {
 
 	logger, _ := Log.NewExtended("standard")
 	contextData := Log.ContextData{TraceID: "trace123", UserID: "external456", Tenant: "tenantABC"}
-	logger.InfoLog("scope", contextData, "some data", "a message")
-	logger.ErrorLog("scope", contextData, errors.New("an error occurred"), "some data", "a message")
-	logger.SuccessLog("scope", contextData, "some data", "a message")
-	logger.DebugLog("scope", contextData, "some data", "a message")
+	logger.Info("scope", contextData, "some data", "a message")
+	logger.Error("scope", contextData, errors.New("an error occurred"), "some data", "a message")
+	logger.Success("scope", contextData, "some data", "a message")
```

Comment:
what is the difference between logger.Success and logger.Out?

Reply:
`logger.Success` is to mark the completion of a function, so you'll see an `ok:`prefix on the log message. the basis of `logger.Success` is `log.Out`